### ISSUE TITLE
Firebase Error when creating a new record

### ISSUE THREAD

**Issue by fabarea** [Camberi/firecms#19]
Steps to reproduce:

1. I have cloned the master of this repository
2. `cd example ; yarn install; yarn start`
3. edit src/firebase_config.ts with my credentials.

As shown by the screenshot, I am able to read / update the records on Firebase but creating a new record generates an exception. 

```
FirebaseError: Function CollectionReference.doc() requires its first argument to be of type non-empty string, but it was: undefined
```

FYI

```
node --version
v14.15.0
```

<img width="2534" alt="image" src="https://user-images.githubusercontent.com/620730/100251600-3b57ff80-2f3f-11eb-86ed-3ecb899d171b.png">

BTW, thank you for this nice package!

**Comment by fgatti675** [comment]
Hey fabarea, thanks for the catch.
It is fixed in the latest release 0.17.2 :)
Btw, any feedback on the library is highly appreciated!

**Comment by fabarea** [comment]
I can confirm this works now fine 👍
